docs/madr/1-intial.md

```diff
@@ -0,0 +1,97 @@
+# Asset resolution by way of an ANT Registry
+
+- Status: accepted
+- Deciders: [Dylan], [Ariel], [Phil]
+- Date: 2024-07-22
+- Authors: [Atticus], [Dylan], [Ariel], [Phil]
+
+## Context and Problem Statement
+
+Performance of loading a user’s registered name is slow with the current method of loading all of the ArNS ecosystem (all ArNS names and ANTs registered to them). There is no way to quickly identify ownership of an ANT, let alone any AO process.
+
+## Decision Drivers
+
+The main drivers for this decision revolve around:
+
+- **Speed**
+  - Uses fewer resources client-side.
+- **Scalability**
+  - We cannot load the entire ArNS ecosystem client-side if it grows.
+- **Ecosystem Integrations**
+  - Having this utility process solely for ANT contract indexing provides integration points for other applications and reduces our reliance on other teams/products (e.g., Bazar profile processes).
+- **Maintainability**
+  - Since we control the registry process, we can drive its maintenance.
+
+## Considered Options
+
+### ANT Registry
+
+- **Pros:**
+  - Dedicated registry for ANTs.
+  - Event-driven updates from ANTs.
+  - Separate state and compute from the resolver process.
+- **Cons:**
+  - Additional maintenance required.
+
+### ArNS Resolver
+
+- **Pros:**
+  - Capable of resolving records and ANTs.
+- **Cons:**
+  - Does not resolve ANTs that are not registered.
+  - Complexity increases with integration of ANT updates.
+
+### Adding Ownership Info to the Network Process
+
+- **Pros:**
+  - Centralized control.
+- **Cons:**
+  - Increases complexity of the Network process.
+  - Reduces flexibility of ANT processes.
+
+### Bazar Profile Processes and Profile Registry
+
+- **Pros:**
+  - Promising technology.
+  - Future integration potential.
+- **Cons:**
+  - Currently in rapid development.
+  - Managed by another team.
+  - Complicates ownership resolution due to account abstraction.
+
+## Decision Outcome
```

Comment:
Nice work here. I feel like I understand the outcome well based on what you've written.